packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,119 @@
+import { Notice } from "@cerner/terra-docs";
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import { Badge } from 'terra-icon/package.json?dev-site-package';
+
+<!-- Examples -->
+import DecorativeIconExample from './example/DecorativeIconExample?dev-site-example';
+import GroupOfIconsExample from './example/GroupOfIconsExample?dev-site-example';
+import InformativeIconExamples from './example/InformativeIconExamples?dev-site-example';
+import FunctionalIconExample from './example/FunctionalIconExample?dev-site-example';
+
+<!-- Icons used in the examples, but also I want to use them in this doc. -->
+import IconManufacturer from 'terra-icon/lib/icon/IconManufacturer';
+import IconAnnouncement from 'terra-icon/lib/icon/IconAnnouncement';
+import IconTile from 'terra-icon/lib/icon/IconTile';
+import IconList from 'terra-icon/lib/icon/IconList';
+import IconTreemap from 'terra-icon/lib/icon/IconTreemap';
+
+<Badge />
+
+# Accessibility Guide for Terra Icon
+
+<Notice variant="important" ariaLevel="2">
+
+Accessibility of icons is important because icons are ubiquitous. They can convey visual meaning or be used as a decorative or supportive element in a UI design and are an essential tool for creating intuitive user experiences that are appropriate for all languages. However, **if implemented improperly or without alternative content, they will not understood everyone**.
+
+</Notice>
+
+Since icons are images, they are not inherently accessible to all users, especially those with significantly impaired vision or blindness. Therefore, interface **icons must have alternative content**.
+
+Screen readers convey alternative content to users who have difficulty interpreting icons. Speech input devices also use alternative content to target specific icons.
+
+It is common for interfaces to use multiples of the same icon on a screen. Development and content teams must consider this use case to ensure all users can predictably understand the icon's purpose.
+
+**Users expect identical icons to perform the same function**. Similarly, screen reader users expect their software to identify identical icons consistently. Therefore, failure to provide consistent identification (visual or alternative content) can cause problems for all users.
+
+In addition, people who use only their voice to interact with technology also rely upon consistent alternative content behind the icons to ensure they can interact with controls.
```

Comment:
Should these be bullet pointed or combine in a paragraph? They currently look like individual sentences separated by a line.

![Screen Shot 2022-03-10 at 1 52 17 PM](https://user-images.githubusercontent.com/22618655/157744923-8d5bdad3-4565-477a-a5a5-6f5f21960d4d.png)

Reply:
I'll make them one paragraph.

Reply:
Fixed in 8be2421ad11674691395463c2e5518e0dfc27aae.